dynamodb-converter/src/test/java/no/unit/nva/model/utils/ExceptionsTest.java

```diff
@@ -0,0 +1,21 @@
+package no.unit.nva.model.utils;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+public class ExceptionsTest {
```

Comment:
I think this can be removed as we should test for the message in the handler test